gulpfile.js

```diff
@@ -121,9 +122,28 @@ gulp.task('copy-test-dirs', function() {
 });
 
 gulp.task('copy-images', function () {
+  var TO_JPEG = {
+    format: 'jpg',
+    formatParams: {
+      quality: 70
+    }
+  };
   return gulp.src(IMG_FILE_TYPES, {
     base: '.'
-  }).pipe(gulp.dest('./dist'));
+  }).pipe(imageConverter({
+    files: {
+      'img/pages/home/hero-unit@2x.png': TO_JPEG,
+      'img/pages/home/hero-unit.png': TO_JPEG,
+      'img/pages/activities/hero-teaching-activities@2x.png': TO_JPEG,
+      'img/pages/activities/hero-teaching-activities.png': TO_JPEG,
+      'img/pages/clubs/hero-clubs@2x.png': TO_JPEG,
+      'img/pages/clubs/hero-clubs.png': TO_JPEG,
+      'img/pages/events/hero-events@2x.png': TO_JPEG,
+      'img/pages/events/hero-events.png': TO_JPEG,
+      'img/pages/teach-like-mozilla/hero-teach-like-mozilla@2x.png': TO_JPEG,
+      'img/pages/teach-like-mozilla/hero-teach-like-mozilla.png': TO_JPEG
+    }
```

Comment:
I wonder if renaming all hero-unit images to `hero-unit.png` and `hero-unit@2x.png` would be an option... so we can just do something like `img/pages/**/hero-unit.png`?  That way we don't have to specify hero-unit images one by one here (I'm worried we might forget to list files here when we add new hero unit images to the site)